.circleci/config.yml

```diff
@@ -25,8 +25,12 @@ jobs:
           command: sudo pip install -r requirements.txt
       
       - run:
-          name: 'Build'
-          command: python -m sphinx -b html -j auto docs docs/_build/html
+          name: 'Build doctest'
+          command: python -m sphinx -b doctest -n -j auto docs docs/_build/html
```

Comment:
The `-n` flag is for "nit-pick", which generates warnings about unresolved references and the like.